CHANGELOG/CHANGELOG-v1.32.md

```diff
@@ -5,20 +5,27 @@
 
  - Add alerts with the recommended course of action to monitor LINSTOR, Piraeus-operator, capacity of storage-pools and resources states
  - Added Grafana dashboard to monitor LINSTOR cluster and DRBD resources
- - By default, all Kubernetes v1.19 clusters will update to v1.21
  - Etcd will be restarted.
+ - For clusters with automatic Kubernetes version selection, Kubernetes v1.21 becomes the default version.
+    * The Kubernetes version update in such clusters will be done during the Deckhouse version update.
+    * Updating the Kubernetes version will cause the restart of the cluster Control Plane components.
+    * Run the following command to find out if the cluster has enabled automatic Kubernetes version selection: `kubectl -n kube-system get secret d8-cluster-configuration -o json | jq '.data."cluster-configuration.yaml"' -r | base64 -d | grep kubernetesVersion`. If the result is ‘kubernetesVersion: Automatic’ — the cluster has enabled automatic Kubernetes version selection.
  - Multimaster clusters will automatically turn LINSTOR into HA-mode
  - Now LVM pools will automatically be added to the LINSTOR cluster and StorageClasses generated
  - OpenVPN will be migrated from using PVC to store certificates to Kubernetes secrets. PVC will still remain in the cluster as a backup. If you don't need it, you should manually delete it from the cluster.
- - This module enables snapshots functionality in Kubernetes cluster
+ - The new module - ceph-csi. Manages the creation of Ceph volumes (RBD and CephFS) and attaches them to workloads.
+ - The new module - snapshot-controller. Enables snapshot support for compatible CSI drivers in a Kubernetes cluster.
```

Comment:
```suggestion
 - The new module - snapshot-controller. Enables snapshot support for compatible CSI drivers and cloud providers in a Kubernetes cluster.
```

Not necessary, but imao can be better for some users who don't know what a csi driver is